sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/MonitorExporterClientTestBase.java

```diff
@@ -39,40 +38,17 @@ AzureMonitorExporterBuilder getClientBuilder() {
             httpClient = interceptorManager.getPlaybackClient();
         }
 
+        List<HttpPipelinePolicy> policies = new ArrayList<>();
+        policies.add(new AzureMonitorRedirectPolicy());
+        policies.add(interceptorManager.getRecordPolicy());
         HttpPipeline httpPipeline = new HttpPipelineBuilder()
             .httpClient(httpClient)
-            .policies(interceptorManager.getRecordPolicy()).build();
+            .policies(policies.toArray(new HttpPipelinePolicy[0]))
```

Comment:
instead of creating policies list above
```suggestion
            .policies(new AzureMonitorRedirectPolicy(), interceptorManager.getRecordPolicy());
```